Formula/fvm.rb

```diff
@@ -0,0 +1,61 @@
+class Fvm < Formula
+    desc "Simple cli to manage Flutter SDK versions per project"
+    homepage "https://fvm.app"
+    url "https://github.com/leoafarias/fvm/archive/2.0.1.tar.gz"
+    sha256 "2a9107963214658eeb7b12af68e8db5274138d978913b38effa682f68041d63e"
+    license "MIT"
+  
+    depends_on "dart-lang/dart/dart" => :build
+  
+    def install
+      dart = Formula["dart-lang/dart/dart"].opt_bin
+      
+      pubspec = YAML.safe_load(File.read("pubspec.yaml"))
+      version = pubspec["version"]
+  
+      # Tell the pub server where these installations are coming from.
+      ENV["PUB_ENVIRONMENT"] = "homebrew:fvm"
+  
+      system dart/"pub", "get"
+      
+      if Hardware::CPU.is_64_bit?
+        # Build a native-code executable on 64-bit systems only. 32-bit Dart
+        # doesn't support this.
+        system dart/"dart2native", "-Dversion=#{version}", "bin/main.dart",
+               "-o", "fvm"
+        bin.install "fvm"
+      else
+        system dart/"dart",
+               "-Dversion=#{version}",
+               "--snapshot=main.dart.app.snapshot",
+               "--snapshot-kind=app-jit",
+               "bin/main.dart", "version"
+        lib.install "main.dart.app.snapshot"
+  
+        # Copy the version of the Dart VM we used into our lib directory so that if
+        # the user upgrades their Dart VM version it doesn't break Sass's snapshot,
+        # which was compiled with an older version.
+        cp dart/"dart", lib
+  
+        (bin/"fvm").write <<~SH
+          #!/bin/sh
+          exec "#{lib}/dart" "#{lib}/main.dart.app.snapshot" "$@"
+        SH
+      end
+      chmod 0555, "#{bin}/fvm"
+    end
+  
+    test do
+      # `test do` will create, run in and delete a temporary directory.
+      #
+      # This test will fail and we won't accept that! For Homebrew/homebrew-core
+      # this will need to be a test that verifies the functionality of the
+      # software. Run the test with `brew test fvm`. Options passed
+      # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+      #
+      # The installed folder is not in the path, so use the entire path to any
+      # executables being tested: `system "#{bin}/program", "do", "something"`.
+      system "false"
```

Comment:
Please include a proper test.

Reply:
Sorry about that, this template was created by a package and that was the default, but I will check.

Reply:
Yes, that's why you're also requested to run `brew audit --new` [*]. If you had run that, then you would have been warned that you need to fix the `test` block.

[*] This will be shown to you by `brew create`. If you didn't use that, it's also in the guidelines for contributing that you marked as having followed.

Reply:
I did read the guidelines and `brew create` seems to be more of a suggestion, `read the Formula Cookbook or: brew create $URL and make edits`. 

Regarding the test, as I mentioned that was a miss on my end, however, I did run `brew audit --strict`, and was getting an exit but it seems it was hanging for a while and exists. (I will take a look to make sure it's solved)

Thanks

Reply:
Thanks @carlocab seems I have some work to do on my end, will close and reopen later as I might have to change some things

Reply:
Please also run `brew audit --new`, as requested [here](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md#to-add-a-new-formula-for-foo-version-234-from-url). This runs extra checks (beyond those with `--strict`) that CI also runs for new formulae.